src/Runners/PHPUnit/ExecutableTest.php

```diff
@@ -175,23 +175,36 @@ public function getLastCommand(): string
         return $this->lastCommand;
     }
 
+    /**
+     * Set the last process command.
+     *
+     * @param string $command
+     */
+    public function setLastCommand(string $command)
+    {
+        $this->lastCommand = $command;
+    }
+
     /**
      * Executes the test by creating a separate process.
      *
-     * @param $binary
-     * @param array $options
-     * @param array $environmentVariables
+     * @param string      $binary
+     * @param array       $options
+     * @param array       $environmentVariables
+     * @param string|null $passthru
+     * @param string|null $passthruPhp
      *
      * @return $this
      */
-    public function run(string $binary, array $options = [], array $environmentVariables = [])
+    public function run(string $binary, array $options = [], array $environmentVariables = [], string $passthru = null, string $passthruPhp = null)
```

Comment:
Imo, the param type declaration needs to be `?string`, otherwise it wouldn't be possible to pass (theoretically, when method is called in PHP code) `null` as `$passthru` and then set `$passthruPhp`.